### ISSUE TITLE
Support adding items to brewing stands

### ISSUE THREAD

**Issue by ghost** [SquidDev/artist#10]
Just to save some time: slots 1-3 are bottles, 4 is ingredient, 5 is blaze powder.

Also, you need to look at item NAMES to see if it's a water bottle or potion, NBT is used for potions.

**Comment by SquidDev** [comment]
Would probably require #2 to be implemented before being useful. We could definitely do auto-refuelling and pickup in the current state, like furnaces do.

**Comment by ghost** [comment]
Yeah, I'd just like to auto put water bottles/magma powder in it.
